src/program-common/eventhivtransmission.cpp

```diff
@@ -181,15 +198,17 @@ double EventHIVTransmission::calculateHazardFactor(const SimpactPopulation &popu
 
 	double Pi = pPerson1->getNumberOfRelationships();
 	double Pj = pPerson2->getNumberOfRelationships();
-
+    	
 	double V = pPerson1->hiv().getViralLoad();
 	assert(V > 0);
 	
 	assert(s_a != 0);
 	assert(s_b != 0);
 	assert(s_c != 0);
 
-	double logh = s_a + s_b * std::pow(V,-s_c) + s_d1*Pi + s_d2*Pj;
+	double logh = s_a + s_b * std::pow(V,-s_c) + s_d1*Pi + s_d2*Pj + s_e1*getH(pPerson1) + s_e2*getH(pPerson2)+ 
+s_g1*pPerson2->hiv().getHazardB0Parameter() +
+s_g2*pPerson2->hiv().getHazardB1Parameter();
 
```

Comment:
Please fix indentation here to improve readability, or even add the last two lines to the `double logh` line